doc/techref/projections.md

```diff
@@ -16,34 +16,34 @@ The table below shows the projection codes for the 31 GMT map projections:
-| **W**[{{ lon0 }}/]*width*                                   | {doc}`Mollweide </projections/misc/misc_mollweide>` |
-| **X***width*[**l**\|**p***exp*\|**T**\|**t**][/*height*[**l**\|**p***exp*\|**T**\|**t**]][**d**] | {doc}`Linear </projections/nongeo/cartesian_linear>`, {doc}`logarithmic </projections/nongeo/cartesian_logarithmic>`, {doc}`power </projections/nongeo/cartesian_power>`, and time |
-| **Y**{{ lon0 }}/{{ lat0 }}/*width*                          | {doc}`Cylindrical equal area </projections/cyl/cyl_equal_area>` |
+| **A**{{ lon0 }}/{{ lat0 }}[/*horizon*]/*width*              | {doc}`/projections/azim/azim_lambert` |
+| **B**{{ lon0 }}/{{ lat0 }}/{{ lat1 }}/{{ lat2 }}/*width*    | {doc}`/projections/conic/conic_albers` |
+| **C**{{ lon0 }}/{{ lat0 }}/*width*                          | {doc}`/projections/cyl/cyl_cassini` |
+| **Cyl_stere**/[{{ lon0 }}/[{{ lat0 }}/]]*width*             | {doc}`/projections/cyl/cyl_stereographic` |
+| **D**{{ lon0 }}/{{ lat0 }}/{{ lat1 }}/{{ lat2 }}/*width*    | {doc}`/projections/conic/conic_equidistant` |
+| **E**{{ lon0 }}/{{ lat0 }}[/*horizon*]/*width*              | {doc}`/projections/azim/azim_equidistant` |
+| **F**{{ lon0 }}/{{ lat0 }}[/*horizon*]/*width*              | {doc}`/projections/azim/azim_gnomonic` |
+| **G**{{ lon0 }}/{{ lat0 }}[/*horizon*]/*width*              | {doc}`/projections/azim/azim_orthographic` |
+| **G**{{ lon0 }}/{{ lat0 }}/*width*[**+a***azimuth*][**+t***tilt*][**+v***vwidth*/*vheight*][**+w***twist*][**+z***altitude*] | {doc}`/projections/azim/azim_general_perspective` |
+| **H**[{{ lon0 }}/]*width*                                   | {doc}`/projections/misc/misc_hammer` |
+| **I**[{{ lon0 }}/]*width*                                   | {doc}`/projections/misc/misc_sinusoidal` |
+| **J**[{{ lon0 }}/]*width*                                   | {doc}`/projections/cyl/cyl_miller` |
+| **Kf**[{{ lon0 }}/]*width*                                  | {doc}`/projections/misc/misc_eckertIV` |
+| **Ks**[{{ lon0 }}/]*width*                                  | {doc}`/projections/misc/misc_eckertVI` |
+| **L**{{ lon0 }}/{{ lat0 }}/{{ lat1 }}/{{ lat2 }}/*width*    | {doc}`/projections/conic/conic_lambert` |
+| **M**[{{ lon0 }}/[{{ lat0 }}/]]*width*                      | {doc}`/projections/cyl/cyl_mercator` |
+| **N**[{{ lon0 }}/]*width*                                   | {doc}`/projections/misc/misc_robinson` |
+| **Oa**{{ lon0 }}/{{ lat0 }}/*azimuth*/*width*[**+v**]       | {doc}`/projections/cyl/cyl_oblique_mercator_1` |
+| **Ob**{{ lon0 }}/{{ lat0 }}/{{ lon1 }}/{{ lat1 }}/*width*[**+v**] | {doc}`/projections/cyl/cyl_oblique_mercator_2` |
+| **Oc**{{ lon0 }}/{{ lat0 }}/{{ lonp }}/{{ latp }}/*width*[**+v**] | {doc}`/projections/cyl/cyl_oblique_mercator_3` |
+| **P***width*[**+a**][**+f**[**e**\|**p**\|*radius*]][**+r***offset*][**+t***origin*][**+z**[**p**\|*radius*]] | {doc}`Polar </projections/nongeo/polar` [azimuthal] ({math}`\theta, r`) (or cylindrical) |
```

Comment:
```suggestion
| **P***width*[**+a**][**+f**[**e**\|**p**\|*radius*]][**+r***offset*][**+t***origin*][**+z**[**p**\|*radius*]] | {doc}`Polar </projections/nongeo/polar>` [azimuthal] ({math}`\theta, r`) (or cylindrical) |
```

Reply:
The link is fixed in commit https://github.com/GenericMappingTools/pygmt/pull/3407/commits/81e161981c272eb47ad3fb1e59d6cce2df53f378.